### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by KengChiLiu** [CVLAB-Unibo/Unsupervised-Adaptation-for-Deep-Stereo#3]
Dear authors, first of all, thanks for the great work.
I'm wondering may you provide your pretrained model if possible?

Thank you

**Comment by AlessioTonioni** [comment]
Hi,
We have added in the two folders inside 'dispflownet-adaptation-release/models/' two 'get_weights.sh' scripts to download pre-trained networks.
+ **DispNet_Adaptation_kitti_train/get_weights.sh** will download the weights of Dispnet fine tuned with our unsupervised loss function on the training set of Kitti2012 using AD-Census for disparity estimation and CCNN as confidence measure for 120000 step.
+ **DispNet_Adaptation_ShadowOnTruck/get_weights.sh** will download the weights of Dispnet fine tuned with our unsupervised loss function on the Shadow on Truck sequence from the [HCI-dataset](https://hci.iwr.uni-heidelberg.de/benchmarks/Challenging_Data_for_Stereo_and_Optical_Flow) using AD-Census for disparity estimation and CCNN as confidence measure.

**Comment by KengChiLiu** [comment]
Thank you so much~
